spec/customer_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'customer'
+require 'dishes'
+
+describe Customer do
+  it { is_expected.to respond_to :check }
+
+  before do
+    @dish = double(:dishes)
```

Comment:
This test double only appears to be used in the test code. So I don't think you need a double. An array would be enough:
```ruby
describe Customer do
  let(:dishes) { ["Chicken = £5", "Noodles = £3", "Steak = £6"] }

  it 'can select a dish from the menu' do
    subject.select(dishes[1])
    ...
```
Please could you remove this double?

Reply:
I have removed this double in the recent push and left it as the array like it was before peer review 👍